### ISSUE TITLE
Is Integration with Angular 12 feasible ? 

### ISSUE THREAD

**Issue by ahadarif007** [devongovett/node-wkhtmltopdf#143]
how to integrate "wkhtmltopdf" with angular 12 ?  or this wrapper is for server side Node js frameworks ?

**Comment by PyroSA** [comment]
Correct, this is a wrapper for software that's installed on the computer.

Since the browser cannot safely run such software from the sandbox, and you generally wouldn't want to force users to install software for a website, you cannot easily use wkhtmltopdf from a browser.

You're likely looking for something like jsPDF.
